src/assets/products.json

```diff
@@ -1,6 +1,7 @@
 [
   {
     "name": "Banana",
+    "available": true,
```

Comment:
I think a `"level": 1` field makes more sense, where `level > currentLevel` implies the product is not yet available.